File: src/main/java/org/jukeboxmc/potion/NightVisionEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class NightVisionEffect extends Effect {

    @Override
    public int getId() {
        return 16;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.NIGHT_VISION;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 0, 0, 139 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/NightVisionEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class NightVisionEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 16
    }

    override fun getEffectType(): EffectType {
        return EffectType.NIGHT_VISION
    }

    override fun getEffectColor(): Color {
        return Color(0, 0, 139)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}